lua/telescope/pickers/layout_strategies.lua

```diff
@@ -344,7 +344,7 @@ layout_strategies.horizontal = make_documented_layout(
     height, h_space = calc_size_and_spacing(height, max_lines, bs, 2, 4, 1)
 
     prompt.height = 1
-    results.height = height - prompt.height - h_space
+    results.height = height - prompt.height - h_space + (self.theme == "minimal" and 1 or 0)
```

Comment:
i think instead we could perhaps have an option that themes pass of whether they want the lines to overlap or not. So you could have an opt for `overlap`, which would apply these changes.

Reply:
oooh I like that idea! would be nice to have the option for all themes

Reply:
I really like this idea too!